sui_programmability/adapter/src/adapter.rs

```diff
@@ -49,40 +53,18 @@ mod adapter_tests;
 // that involve re-computation / deserialization to otherwise get.
 pub struct SuiMoveVM {
     movevm: MoveVM,
```

Comment:
Please add a TODO here to figure out whether we still need `SuiMoveVM` latter.
This was added because we needed the cache. But if all we need is the movevm, maybe we don't need this layer.